### PR TITLE
Min disk

### PR DESCRIPTION
**Pull request by akarukappadath**
https://github.com/DataBiosphere/leonardo/issues/388

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [x] Get two thumbsworth of review and PO signoff if necessary
- [x] Verify all tests go green
- [x] Squash and merge; you can delete your branch after this
- [x] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20180607_master_disk_size.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (rtitle) [inline src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20180607_master_disk_size.xml]: I think this file needs to be referenced here to actually run:

https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml

File: src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -721,7 +721,7 @@ definitions:
       masterDiskSize:
         type: integer
         description: |
-          Optional, the size of the disk on the master node. Minimum size is 100GB. If unspecified, default size is 500GB.
+          Optional, the size of the disk on the master node. Minimum size is 10GB. If unspecified, default size is 100GB.
```
Comment (rtitle) [inline src/main/resources/swagger/api-docs.yaml]: Nitpick: can we say "the size in gigabytes of the disk". I feel like it's better to specify the units.

File: src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala [hunk 3]
```diff
@@ -179,11 +179,11 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
   }
 
   it should "create a standard cluster with 2 workers and override too-small disk sizes with minimum disk size" in isolatedDbTest {
-    val machineConfig = MachineConfig(Some(2), Some("test-master-machine-type"), Some(50), Some("test-worker-machine-type"), Some(10), Some(3), Some(4))
+    val machineConfig = MachineConfig(Some(2), Some("test-master-machine-type"), Some(5), Some("test-worker-machine-type"), Some(5), Some(3), Some(4))
```
Comment (rtitle) [inline src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala]: 👍 , this seems like an adequate test

**Comment by akarukappadath** [comment]
jenkins retest

**Pull request merged**